CMakeLists.txt

```diff
@@ -102,22 +106,44 @@ if(TARGET_PLATFORM STREQUAL "linux")
 endif()
 
 # Default values
-set(BEEROCKS_WLAN1_IFACE "wlan0")
-set(BEEROCKS_WLAN2_IFACE "wlan2")
-set(BEEROCKS_WLAN3_IFACE "wlan4")
-set(BEEROCKS_BRIDGE_IFACE "br-lan")
+if(NOT DEFINED BEEROCKS_WLAN1_IFACE)
+    set(BEEROCKS_WLAN1_IFACE "wlan0")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN2_IFACE)
+    set(BEEROCKS_WLAN2_IFACE "wlan2")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN3_IFACE)
+    set(BEEROCKS_WLAN3_IFACE "wlan4")
+endif()
+if(NOT DEFINED BEEROCKS_BRIDGE_IFACE)
+    set(BEEROCKS_BRIDGE_IFACE "br-lan")
+endif()
```

Comment:
All this stuff really should be removed. Instead, we should parse it out of the configuration file, if appropriate. For prplmesh_utils.sh that's actually a little difficult... So OK for now.

Reply:
I used cached variable for them too to have 4 lines instead of 12 as here